### PR TITLE
[es] Translated subtraction assignment page

### PR DESCRIPTION
**Pull request by fedeperin**
### Description

Translated the subtraction assignment page to Spanish.

### CODE REVIEW

**Comment by fedeperin** [comment]
Hello,
I don't know why the "Lint and review content files / lint-and-review-docs (pull_request_target)" step is failing, I tried to solve all the problems mentioned by the github actions bot, but it still throws the error. Can somebody please help me solving this?

**Comment by Graywolf9** [comment]
Hi! Thank you so much for your contrution, I'll check the error, sometimes the messages don't show much info

File: files/es/web/javascript/reference/operators/subtraction_assignment/index.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+{{Specifications}}
+
+## Compatibilidad con navegadores
+
+{{Compat}}
+
+## Véase también
+
+- [Assignment operators in the JS guide](/es/docs/Web/JavaScript/Guide/Expressions_and_operators#assignment_operators)
+- [Subtraction (`-`)](/es/docs/Web/JavaScript/Reference/Operators/Subtraction)
```
[the remark this answers is not in the document]
Comment (Graywolf9) [reply files/es/web/javascript/reference/operators/subtraction_assignment/index.md:47]: This is the message it should be showed: 
```
files/es/web/javascript/reference/operators/subtraction_assignment/index.md:47:78 MD047/single-trailing-newline Files should end with a single newline character
```

File: files/es/web/javascript/reference/operators/subtraction_assignment/index.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
```
Comment (Graywolf9) [inline files/es/web/javascript/reference/operators/subtraction_assignment/index.md:47]: Missing translations

```suggestion
- [Operadores de asignación en la guía JS](/es/docs/Web/JavaScript/Guide/Expressions_and_operators#assignment_operators)
- [Sustracción (`-`)](/es/docs/Web/JavaScript/Reference/Operators/Subtraction)
```

**Review by Graywolf9 — APPROVED** [review]
Thank you so much for all your time and contribution!

**Pull request merged**